ftests/core/src/main/java/org/commonjava/indy/ftest/core/content/AbstractMetadataTimeoutWorkingTest.java

```diff
@@ -29,6 +29,11 @@
 import static org.hamcrest.CoreMatchers.notNullValue;
 import static org.junit.Assert.assertThat;
 
+/**
+ * This abstract class will do some preparation work for testing of the metadata content timeout function,
+ * like prepare mock remote with a http server, set up test repo, prepare normal content and metadata content.
+ *
+ */
```

Comment:
Wow, thanks for going back and adding these in the last batch of functional tests. :smile: